staging/src/k8s.io/apiserver/pkg/apis/apiserver/types.go

```diff
@@ -147,3 +147,36 @@ type TLSConfig struct {
 	// +optional
 	ClientCert string
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// OpenTelemetryClientConfiguration provides versioned configuration for opentelemetry clients.
```

Comment:
Again is this something we can change later, or should we just use the TracingClientConfiguratio name now?

Reply:
Agreed.  Changed to `TracingConfiguration`.